x-pack/plugins/apm/common/es_fields/apm.ts

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
+export const TIMESTAMP = 'timestamp';
```

Comment:
Shouldn't this be `@timestamp`?

Reply:
`@timestamp` returns the date as string, I'd have to loop through the items to convert to `us` since this is the unit used in the waterfall.
 
`timestamp.us` I wanted to keep consistency since we return the `duration.us`, and IMHO it's good to make it clear what unit it is formatted.

Reply:
I'm not following entirely what you say so maybe I'm missing something. It sounds like you want the timestamp in microseconds in which case you want `timestamp.us`, no?
`timestamp` itself is not a field, but an object containing a single field `us`. I think we should stick to adding fields, not objects to this file.

Reply:
you're right.